addons/compat_3as/CfgVehicles.hpp

```diff
@@ -0,0 +1,47 @@
+class CfgVehicles {
+    class Helicopter_Base_H;
+    class 3as_laat_Base: Helicopter_Base_H {
+        GVARMAIN(passengerTurrets)[] = {

```

Comment:
Are these correct?
They just look like the LS ones, 3AS probably has different set of turret paths